data-pipeline/src/pipeline/main.py

```diff
@@ -353,14 +353,15 @@ def pre_process_bfr(run_type, year):
                     usecols=[
                         "TrustUPIN",
                         "EFALineNo",
+                        "Y1P2",
                         "Y2P2",
                     ],
                 )
                 .rename(
                     {"TrustUPIN": "Trust UPIN"},
                     axis=1,
                 )
-                .query("EFALineNo == 430")
+                .query("EFALineNo in (430, 999,)")
             )
 
     academies_y1 = None
```

Comment:
@Chris-Lorch, IIRC when we first looked at this change we left the academies/SOFA join in place to retain the "Company Registration Number" which isn't in the latter).

However, following these historical data through: do we actually use that column from these y1/y2 DataFrames? Probably out of scope for this fix but this whole section could be simplified somewhat if that's the case.